geomstats/geometry/hyperbolic.py

```diff
@@ -881,6 +882,29 @@ def dist(self, point_a, point_b):
             raise NotImplementedError(
                 'dist is only implemented for ball and extrinsic')
 
+    def squared_dist(self, point_a, point_b):
+        """Geodesic squared hyperbolic distance.
+
+        Redefinition the squared distance implemented
```

Comment:
Override the squared distance ... .

I don't understand why you need to override this method. The squared dist from RiemannianMetric is not called?